docs/containers/quickstart-python.md

```diff
@@ -0,0 +1,142 @@
+4. Select either **Yes** or **No** when prompted to include [Docker Compose](https://docs.docker.com/compose/) files.
+
+5. Enter the relative path to the app’s entry point. This excludes the workspace folder you start from. According to [official Django documentation](https://docs.djangoproject.com/en/3.0/intro/tutorial01/#creating-a-project), this path is commonly `manage.py` (root folder) or `subfolder_name/manage.py`. According to [official Flask documentation](https://flask.palletsprojects.com/en/1.1.x/api/), this is the path to where you create your Flask instance.
+
+>**Tip**: You may also enter the path to a folder name as long as this folder includes a `__main__.py` file.
+
+6. If **Python: Django** or **Python: Flask** was selected, specify app port for local development. Django defaults to port 8000 while Flask defaults to port 5000, however, any unused port will work.
+
+7. With all this information, the Docker extension creates the following files:
+
+    - A `Dockerfile`. To learn more about Intellisense in this file, refer to the [overview](/docs/containers/overview.md).
+
+    - A `.dockerignore` file to reduce the image size by excluding files and folders that aren't needed such as `.git`, `.vscode`, and `__pycache__`.
+
+    - If Docker Compose was selected, a `docker-compose.yml` file.
+
+    - If one does not already exist, a `requirements.txt` file for capturing all app dependencies.
+    > **Tip**: If the virtual environment/host machine is meant to be identical to the container, ensure app dependencies are ported over by running `pip freeze > requirements.txt` in the terminal to overwrite this file.
+
+### File modifications for Django/Flask apps
+
+To give Python Web Developers a great starting point, we chose to use [Gunicorn](https://gunicorn.org/#docs) as the default web server. Since it is referenced in the default Dockerfile, it is included as a dependency in the `requirements.txt` file.
+
+> **Note**: To use Gunicorn as your web server, it must be included in the `requirements.txt` file as an app dependency. It does not need to be installed in your virtual environment/host machine.
+
+#### Django Apps
+
+To use Gunicorn, it must bind to an application callable as an entry point. This callable is declared in the `wsgi.py` file of a Django application. To accomplish this binding, the final line in the Dockerfile says:
+
+```docker
+CMD ["gunicorn", "--bind", "0.0.0.0:8000", "{workspace_folder_name}.wsgi"]`
+```
+
+This assumption is based off Django's default project structure. If your project does not follow `django_project_name (workspace folder) -> django_project_name (subfolder) -> wsgi.py`, you must overwrite the Gunicorn entry point in the Dockerfile to locate the correct `wsgi.py` file.
+
+> **Tip**: If your `wsgi.py` file is in the root folder, the final argument in the command above will be `"wsgi"`. Within subfolders, the argument would be `"subfolder1_name.subfolder2_name.wsgi"`.
```

Comment:
```suggestion
> **Tip**: If your `wsgi.py` file is in the root folder, the final argument in the command above will be `wsgi`. Within subfolders, the argument would be `subfolder1_name.subfolder2_name.wsgi`.
```

Reply:
In this suggestion, the user will actually need the double quotations "" around their words. Do you still believe I should change it?

Reply:
Personally I would like to keep it as is for clarity

Reply:
Please keep it if they need it, sorry!